blocks/header/header.css

```diff
@@ -476,6 +482,367 @@ header .megamenu>.section-outer {
+
+    a {
+      font-family: var(--body-font-family);
+    }
+
+    > .section > .default-content-wrapper {
+      display: flex;
+      flex-direction: column;
+      gap: 25px;
+
+      /* add 25px padding to the top of the second element */
+      > :nth-child(2) {
+        padding-top: 25px;
+      }
+
+      >p, >p a {
+        display: flex;
+        gap: 10px;
+        color: inherit;
+      }
+
+      /* make flex direction column for the last p */
+      >p:last-child {
+        flex-direction: column;
+        gap:0;
+        border: 2px solid transparent;
+
+        br {
+          display: none;
+        }
+
+        >a {
+          text-decoration: none;
+          color: white;
+        }
+
+        /* first child bolder */
+        >a:first-child {
+          font-weight: 600;
+        }
+      }
+    }
+  }
+
+  .megamenu> :first-child:not(:only-child) {
+    border-bottom: 1px solid #EBECF5;
+    margin-bottom: 1.5em;
+  }
+
+  .megamenu {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    width: 100%;
+
+    .section-outer {
+      display: flex;
+      flex-direction: column;
+
+      .section:not(.content-cards)> :first-child:not(:only-child) {
+        border-bottom: 1px solid #EBECF5;
+        margin-bottom: 1.5em;
+      }
+
+      .section {
+        display: flex;
+        flex-direction: column;
+
+        .accordion details {
+          border: none;
+
+          & p {
+            width: 100%;
+          }
+        }
+
+        .accordion details summary::after {
+          content: url("/icons/solutions-right.svg");
+          top: 50%;
+          transform: translate(0, -50%) scale(0.45) rotate(90deg);
+          right: 10px;
+        }
+
+        .accordion details[open] summary::after {
+          transform: translate(0, -50%) scale(0.45) rotate(-90deg);
+        }
+
+        .accordion details[open] .accordion-item-body {
+          transition: none;
+        }
+
+        .accordion .accordion-item-body {
+          .button-container {
+            margin: 0;
+
+            &>* {
+              width: 100%;
+              display: block;
+
+              .button {
+                width: 100%;
+              }
+            }
+          }
+
+          .sub-content {
+            padding: 20px;
+            border-left: 3px solid var(--berry);
+
+            .icon {
+              height: 30px;
+              padding-right: 10px;
+            }
+
+            a {
+              display: flex;
+              color: inherit;
+              text-decoration: underline;
+              padding-left: 20%;
+
+              &::before {
+                content: url("/icons/icon-arrow-right.svg");
+                position: relative;
+                display: block;
+                height: 20px;
+                padding-right: 10px;
+              }
+            }
+          }
+        }
+
+      }
+
+      .section>* {
+        width: 95% !important;
```

Comment:
This is making the linked images at the bottom be out of proportion. 

![image](https://github.com/user-attachments/assets/67085599-34ca-4971-8afb-f5d3d51d26e6)